docusaurus/docs/Android/04-compose/03-channel-components/04-default-channel-item.mdx

```diff
@@ -0,0 +1,203 @@
+        ChannelDetails(
+            channel = it,
+            currentUser = currentUser,
+            modifier = Modifier
+                .weight(1f)
+                .wrapContentHeight()
+        )
+    },
+    trailingContent: @Composable RowScope.(Channel) -> Unit = {
+        ChannelLastMessageInfo(
+            channel = it,
+            currentUser = currentUser,
+            modifier = Modifier
+                .padding(horizontal = 8.dp)
+                .wrapContentHeight()
+                .align(Alignment.Bottom)
+        )
+    },
+    divider: @Composable ColumnScope.() -> Unit = {
+        Spacer(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(0.5.dp)
+                .background(color = ChatTheme.colors.borders)
+        )
+    },
+)
+```
+
+* `modifier`: The modifier parameter for the root component. You can apply a background, elevation, padding, shape, touch handlers and much more.
+* `leadingContent`: Customizable composable function that allows you to override the content that appears at the start of the list item. By default, it represents a `ChannelAvatar`.
+* `detailsContent`: Customizable composable function that allows you to override the center part of the list item. By default, it represents the `ChannelDetails` that show its name and the last message preview.
+* `trailingContent`: Customizable composable function that allows you to override the content that appears at the end of the list item. By default, it represents the `ChannelLastMessageInfo` that shows the unread message indicator if there are unread messages and the last message read state.
+* `divider`: Customizable composable function that allows you to override the content that appears at the bottom of the list item, as a divider.
+
+Here's a simple example for building your own channel item, by overriding the `mentioned` parameters:
+
+```kotlin
+override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+
+    setContent {
+        val user by listViewModel.user.collectAsState() // fetch user
+
+        ChatTheme {
+            ChannelList(
+                itemContent = { channel -> // Customize the channel items
+                    CustomChannelListItem(channel = channel, user = user)
+                }
+            )
+        }
+    }
+}
+
+@Composable
+fun CustomChannelListItem(channel: Channel, user: User?) {
+    DefaultChannelItem(
+        channel = channel,
+        currentUser = user,
+        onChannelLongClick = { },
+        onChannelClick = { },
+        divider = { // Override the divider
+            Spacer(
+                modifier = Modifier
+                    .fillMaxWidth(0.5f)
+                    .height(0.5.dp)
+                    .align(CenterHorizontally)
+                    .background(color = ChatTheme.colors.textLowEmphasis)
+            )
+        },
+        trailingContent = { // Replace the last message content with a spacer
```

Comment:
Last message content is a bit misleading name, as the last message preview is actually in the `ChannelDetails` component.

Reply:
Yeah that's good, I'll update to "trailing content"